docs/pages/versions/unversioned/sdk/symbols.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Symbols
+description: A library that allows access to native symbols.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-symbols'
+packageName: 'expo-symbols'
+iconUrl: '/static/images/packages/expo-symbols.png'
+platforms: ['ios']
+---
+
+import APISection from '~/components/plugins/APISection';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+
+> **warning** This is library is currently in beta and subject to breaking changes.
+
+**`expo-symbols`** provides access to the [SFSymbols](https://developer.apple.com/sf-symbols/) library on iOS.
```

Comment:
```suggestion
`expo-symbols` provides access to the [SFSymbols](https://developer.apple.com/sf-symbols/) library on iOS.
```

Reply:
I see in Apple docs that the library name is referenced as "SF Symbols" instead of "SFSymbols". Should we also use the same pattern here and the rest of the doc where we mention this library name (that usually links to Apple's docs/resources)?

Not something to block this PR but we should avoid introducing new/altered patterns of our own for stuff that are already defined in platform's documentation.